### DISCUSSION TITLE
I need an admin accept my sign up

### DISCUSSION THREAD

**Discussion by Emarcayata** [biigle/core#767] [Q&A] [answered]
Hi! The last weeks I was trying to use BIIGLE. I just could annotate images as a member of another projects. However, I couldn't create my own project or label tree, due to the advertisement: 
You will be able to create new projects and label trees once an admin has accepted your sign up.
Please help me!
Kind regards, 
Esther :)

**Comment by mzur** [comment, marked as answer]
Hi, you signed up with a missing or incomplete affiliation. Usually, we inform you via email about this but in your case, the email was not sent, sorry. You should have received a new email by now and we can resolve the issue there.
